### ISSUE TITLE
[BUG] tox 3.8+ prints its own traceback when a test fails

### ISSUE THREAD

**Issue by seanh** [tox-dev/tox#1228]
Since upgrading to tox 3.8 whenever a test fails I'm seeing an `InvocationError` traceback from tox, as well as the pytest output. The tox traceback seems to be printed to stderr:

```
========================================================= FAILURES ==========================================================
______________________________________________ test_init_sets_given_attributes ______________________________________________

    def test_init_sets_given_attributes():
        group = models.Group(name="My group", authority="example.com", enforce_scope=False)
>       assert False
E       assert False

group      = <Group: my-group>

tests/h/models/group_test.py:22: AssertionError
========================================== 1 failed, 3503 passed in 78.03 seconds ===========================================
Traceback (most recent call last):
  File "/usr/local/bin/tox", line 11, in <module>
    sys.exit(cmdline())
  File "/usr/local/lib/python2.7/dist-packages/tox/session/__init__.py", line 42, in cmdline
    main(args)
  File "/usr/local/lib/python2.7/dist-packages/tox/session/__init__.py", line 63, in main
    retcode = session.runcommand()
  File "/usr/local/lib/python2.7/dist-packages/tox/session/__init__.py", line 177, in runcommand
    provision_tox(provision_tox_venv, self.config.args)
  File "/usr/local/lib/python2.7/dist-packages/tox/session/commands/provision.py", line 9, in provision_tox
    action.popen(provision_args, redirect=False)
  File "/usr/local/lib/python2.7/dist-packages/tox/action.py", line 121, in popen
    raise InvocationError(cmd_args_shell, exit_code, output)
tox.exception.InvocationError: InvocationError for command /home/seanh/Projects/h/.tox/.tox/bin/python -m tox -qqq (exited with code 1)
```

This is [my `tox.ini` file](https://github.com/hypothesis/h/blob/master/tox.ini) and the command I'm running is simply `tox` (or `tox -qqq`).

Here's the [full output of `tox -rvv tests/h/models/group_test.py::test_init_sets_given_attributes`](https://gist.github.com/seanh/301c50e4c8a88251f611fc273696ae94) (both stdout and stderr).

**Comment by gaborbernat** [comment]
Looking at the log seems we did not suppress the auto-provisioned (meta) tox raised exception. Can you try if you still observe the issue if you install ``tox-pip-extensions``?

**Comment by gaborbernat** [comment]
@seanh an FYI tox is written always with a lowercase t 👍

**Comment by seanh** [comment]
Yup, if I install tox-pip-extensions globally the traceback goes away

**Comment by gaborbernat** [comment]
We released tox 3.8.3 that should now fix it 👍
